app/abilities/atmosphere/appliance_ability_builder_ext.rb

```diff
@@ -0,0 +1,9 @@
+module Atmosphere::ApplianceAbilityBuilderExt
+  extend ActiveSupport::Concern
+
+  private
+
+  def can_start_ext?(appl)
```

Comment:
Unused method argument - `appl`. If it's necessary, use `_` or `_appl` as an argument name to indicate that it won't be used. You can also write as `can_start_ext?(*)` if you want the method to accept any arguments but don't care about them.